packages/docs/src/components/Search.jsx

```diff
@@ -1,16 +1,14 @@
 import { useEffect, useState } from 'react'
-import { OramaClient } from '@oramacloud/client'
-import { SearchBox, SearchButton } from '@orama/searchbox'
-import '@orama/searchbox/dist/index.css'
+import { OramaSearchBox, defineCustomElements } from '@orama/react-components'
+import '@orama/react-components/dist/index.css'
```

Comment:
style should be already bundled actually, if you remove this import I expect you to not have problems with style